app/controllers/idv/verify_document_controller.rb

```diff
@@ -0,0 +1,85 @@
+module Idv
+  class VerifyDocumentController < ApplicationController
```

Comment:
Can we add a class comment for this controller? For future us, when we want to know what the difference between `Idv::ImageUploadsController` and `Idv::VerifyDocumentController` is

Also should this controller be plural?

Reply:
good ideas. you think i should add API to the name also?

Reply:
We haven't put API in controller names so far, so I don't feel that strongly about it